docs/operations/export_api.md

```diff
@@ -0,0 +1,425 @@
+# Export API
+
+## Prerequesites
+None
+
+### Export Feature Gate
+
+VMExport support must be enabled in the feature gates to be supported. The
```

Comment:
How about:

VMExport must be enabled in the feature gates for this feature to be available

There is no "support" for any features in an open source project.

Reply:
Done